test/connector/tcp/mssql/secretless-debug/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.0'
+
+services:
+
+  mssql:
+    image: mcr.microsoft.com/mssql/server:2017-latest
+    ports:
+      - 1433:1433
+    environment:
+      # This hardcoded password must match the one in secretless.yml.
+      SA_PASSWORD: "yourStrong()Password"
+      ACCEPT_EULA: Y
+
+  secretless-debug:
+    build:
+      context: ../../../../..
+      dockerfile: Dockerfile.debug
+    ports:
+      - 2223:2223
+      - 40000:40000
+    security_opt:
+      - apparmor:unconfined
+      - seccomp:unconfined
+    cap_add:
+      - SYS_PTRACE
+    volumes:
+      - ../secretless.yml:/secretless.yml
+    depends_on:
+      - mssql
```

Comment:
why can't we just add this image to the `test/connector/tcp/mssql` docker-compose.yml file and update the README in that dir to include these instructions? it seems not very DRY to me to have a separate dir within the MSSQL test suite for this stuff

Reply:
Yes, good point! I moved the secretless-debug docker-compose stuff into the parent directory, and eliminated duplication in the test-local scripts. As for the README, there's no README in the parent. Should I move it up and add some notes for generic secretless testing?